pkg/sql/opt/exec/execbuilder/scalar_builder.go

```diff
@@ -185,6 +186,14 @@ func (b *Builder) buildCast(ctx *buildScalarCtx, ev xform.ExprView) tree.TypedEx
 	return expr
 }
 
+func (b *Builder) buildCoalesce(ctx *buildScalarCtx, ev xform.ExprView) tree.TypedExpr {
```

Comment:
What about tests for COALESCE?

Reply:
Ah, I meant to add some but it slipped my mind. Will open another PR